elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/timegrains/serde/DaySerdeTest.java

```diff
@@ -59,4 +61,14 @@ public void testDeserializeDateInvalidFormat() throws ParseException {
             serde.deserialize(dateInString);
         });
     }
+
+    @Test
+    public void testISODateString() throws ParseException {
+        String dateInString = "2020-01-01T00:00:00-0500";
+        Day expectedDate = new Day(formatter.parse(dateInString));
```

Comment:
This should be isoFormatter instead of formatter. Also might be a good idea to update dateInString to have different hour minutes.